terms/verifiable_secret_sharing.md

```diff
@@ -0,0 +1,81 @@
+---
+comments: true
+---
+# Verifiable Secret Sharing
+
+## Distributed Key Generation
+
+Read the definition of DKG [here](./distributed_key_generation.md).
+
+## Verifiable Secret Sharing (VSS)
+
+### Definition
+
+A $(t, n)$ VSS includes two phases: *Share* and *Reconstruct* as follow:
```

Comment:
```suggestion
A $(t, n)$ VSS includes two phases: *Sharing* and *Reconstruction* as follow:
```

Reply:
I followed the definition in Orochi cookbook:
https://docs.orochi.network/dkg/verifiable-secret-sharing/syntax.html